README.md

```diff
@@ -63,86 +62,81 @@ Must be a list of *tensor specification keys*.
 Please use an image smaller than 500KB, aspect ratio width to height 2:1. The supported image formats are: `jpg`, `png`, `gif`.
 <!--- `I am not quite sure what we decided on for the uri identifiers in the end, I am sticking with the simplest option for now <format>+<protocoll>://<path>`, e.g.: `conda+file://./req.txt` -->  
+
 - `dependencies` Dependency manager and dependency file, specified as `<dependency manager>:<relative path to file>`\
 For example:
   - conda:./environment.yaml
   - maven:./pom.xml
   - pip:./requirements.txt
 
-- `weights`
-A list of weights, each weights definition contains the following fields:
-    - `id` a unique id which will be used to refer to the weights. <!-- maybe with special values like 'default'? -->
-    - `name` the name of the weights for display, it should be a human-friendly name in title case
-    - `description` description about the weights, it is recommended to describe the how the weights is trained, and what's the dataset used for training.
-    - `authors` a list of authors. This field is optional, only required if the authors are different from the model.
-    - `covers` a list of cover images (see `model`:`covers`). This is used for showing how inputs and outputs look like with this weights file.
-    - `source` link to the weights file. Preferably an url to the weights file.
-    - `sha256` SHA256 checksum of the model weight file specified by `source` (see `models` section above for how to generate SHA256 checksum)
-    - `timestamp` timestamp according to [ISO 8601](#https://en.wikipedia.org/wiki/ISO_8601)
-    - `test_inputs` list of URIs to test inputs as described in `inputs` for a single test case. Supported file formats/extensions: `.npy`
-    - `test_outputs` analog to `test_inputs`.
-    - `documentation` relative path to file with additional documentation in markdown.
-    - `tags` a list of tags.
-    - `attachments` text keys and URI values to additional, relevant files.
+- `attachments` Additional files for this specification; e.g. images that are necessary for the documentation.
+
+- `test_inputs` list of URIs to test inputs as described in inputs for a single test case. Supported file formats/extensions: .npy
+- `test_outputs` analog to test_inputs.
 
+- `weights` The weights for this model. Weights can be given for different formats, but should otherwise be equivalent.
+   - `weights_format` Format of this set of weights
+        - `authors` a list of authors. This field is optional, only required if the authors are different from the authors specified in root.
+        - `source` link to the weights file. Preferably an url to the weights file.
+        - `sha256` SHA256 checksum of the model weight file specified by `source` (see `models` section above for how to generate SHA256 checksum)
+        - `timestamp` timestamp according to [ISO 8601](#https://en.wikipedia.org/wiki/ISO_8601)
```

Comment:
Would be nice to add `attachments` to allow weights specific attachments for the model packaging, e.g. when we generate a model package with only one weights formats, we will need to know what files to include for this package.

And also perhaps we should mention which field is optional (applies also to the entire doc).

Reply:
Ok, that use case makes sense. I will add it back here and declare it optional.
I will also try to see if I can spot some other cases where we forgot to mention that it's optional.